.github/workflows/nightly-tests.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         uses: ./.github/actions/integration-test
         with:
           test-server: workspace-wsmtest.json
-          test: suites/FullIntegration.json
+          test: suites/wsmtest/FullIntegration.json
```

Comment:
Why do these need to be put into a subdirectory as opposed to being renamed in place?
We need a different name. `wsmtest` is a personal environment, not a kind of test suite. These tests are run in may environments. Not just wsmtest.

Reply:
It is even stranger, because you have configs for dev, alpha, staging, but you add wsmtest for suites?!
I think this scheme is unmaintainable.